analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -80,7 +80,7 @@ abstract class AbstractIntegrationSpec : StringSpec() {
         // Do not use the usual simple class name as the suffix here to shorten the path which otherwise gets too long
         // on Windows for SimpleFormIntegrationTest.
         outputDir = createTempDir(ORT_NAME)
-        downloadResult = Downloader().download(pkg, outputDir)
```

Comment:
commit-message: Could you add a rationale why you are doing that?

Reply:
Done.